src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private enum Colors {
+        WHITE,
+        BLACK,
+        RED,
+        ORANGE,
+        YELLOW,
+        GREEN,
+        BLUE,
+        PURPLE
+    }
```

Comment:
you can make it like a nested class but better move it after public  and private methods ([read about recommended class content ordering ](https://mate-academy.github.io/style-guides/java/java.html#s3.4-class-declaration))